app/static/ie-status.json

```diff
@@ -3904,5 +3904,28 @@
     "demo": "",
     "id": 5608754573737984,
     "uservoiceid": null
+  },
+      {
+    "name": "International input[type=email] validation",
+    "category": "HTML5",
+    "link": "https://www.w3.org/Bugs/Public/show_bug.cgi?id=15489",
+    "summary": "Adding support for international email addresses that use UTF-8 characters (e.g. 伊昭傑@郵件.商務, राम@मोहन.ईन्फो)",
+    "standardStatus": "Public discussion",
+    "ieStatus": {
+      "text": "Under Consideration",
+      "iePrefixed": "",
+      "ieUnprefixed": ""
+    },
+    "impl_status_chrome": "",
+    "ff_views": {
+      "text": "Shipped",
+      "value": 1
+    },
+    "shipped_opera_milestone": false,
+    "msdn": "",
+    "wpd": "",
+    "demo": "",
+    "id": 2772800,
```

Comment:
I think you want to leave the id field off. This field is only necessary if we're pulling data from chromestatus.com.  This feature doesn't appear to be on chromestatus, so you can leave this off.